controller_wr_driver/include/uc_pc_link_defs.h

```diff
@@ -0,0 +1,66 @@
+#ifndef UC_PC_LINK_DEFS_H_
+#define UC_PC_LINK_DEFS_H_
+
+/*** Input data ***/
+/*
+ * Velocity command data
+ * Data:
+ *  float[0] = velocity [mps]
+ *  float[1] = steering [deg]
```

Comment:
Don't understand this comment

Reply:
We shouldn`t change the value - this file is prototype document for usage in all applications that are connected by one object .